### PR TITLE
[FIX] another overlooked edgecase

### PR DESCRIPTION
**Pull request by tony-sappe**
**Description:**
Found another edge case during situations the loader is run with no delete records to process

**Technical details:**
N/A

**Requirements for PR merge:**

1. [x] Unit & integration tests updated (N/A)
2. [x] API documentation updated (N/A)
3. [x] Necessary PR reviewers:
    - [x] Backend
4. [x] Matview impact assessment completed (N/A)
5. [x] Frontend impact assessment completed (N/A)
6. [x] Data validation completed
7. [x] Appropriate Operations ticket(s) created (N/A)
8. [x] Jira Ticket [DEV-1671](https://federal-spending-transparency.atlassian.net/browse/DEV-1671):
    - [x] Link to this Pull-Request
    - [x] Performance evaluation of affected Script
    - [x] Before / After data comparison

**Area for explaining above N/A when needed:**
```
Edit to ETL script code
```

### CODE REVIEW

File: usaspending_api/broker/management/commands/fabs_nightly_loader.py [hunk 1]
```diff
@@ -152,8 +152,8 @@ def handle(self, *args, **options):
         is_incremental_load = not any((reload_all, afa_ids, start_datetime, end_datetime))
 
         if is_incremental_load:
-            last_load_date = get_last_load_date()
-            logger.info("Processing data for FABS starting from %s" % last_load_date)
+            start_datetime = get_last_load_date()
+            logger.info("Processing data for FABS starting from %s" % start_datetime)
```
Comment (BrianZito) [inline usaspending_api/broker/management/commands/fabs_nightly_loader.py]: personally I feel like "start_datetime" sounds like the time at which the load started running, but I'm not holding up a PR for that.

Comment (tony-sappe) [reply usaspending_api/broker/management/commands/fabs_nightly_loader.py]: Perhaps. But I'll leave as-is since it is a pre-existing parameter and variable
